packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json

```
@@ -0,0 +1,394 @@
+{
+  "Description": "The CDK Toolkit Stack. It was created by `cdk bootstrap` and manages resources necessary for managing your Cloud Applications with AWS CDK.",
+  "Parameters": {
+    "TrustedPrincipals": {
+      "Description": "List of AWS principals that the publish and action roles should trust to be assumed from",
```

The description is unclear here. Maybe:

```suggestion
      "Description": "List of AWS principals that are trusted to publish assets and deploy stacks to this environment",
```